Training/PythonWebServer/index.html

```diff
@@ -9,6 +9,89 @@
     <script src="/resources/index.js" type="text/javascript"></script>
 </head>
 <body>
-    <h1>Hello world!</h1>
+	<div class='wrapper'>
+        <button class="button" onclick="loadDoc()">UPDATE</button>
+
+    	<div class="display">
+        Time:
+            <p id = "Time1"></p> 
+            <p id = "Time2"></p> 
+            <p id = "Time3"></p> 
+            <p id = "Time4"></p>
+            <p id = "Time5"></p> 
+            <p id = "Time6"></p> 
+            <p id = "Time7"></p> 
+            <p id = "Time8"></p>
+        <br><br><br>
+    	</div>
+
+    	<div class="display">
+        Total CPUs: 
+        <p id = "totalCPU"></p>
+        <br> <br>
+    	CPU Usage:
+            <p id = "cpuUsage1"></p> 
+            <p id = "cpuUsage2"></p> 
+            <p id = "cpuUsage3"></p> 
+            <p id = "cpuUsage4"></p>
+    	</div>
+
+    	<div class="display">
+    	Total RAM:
+            <p id = "totalRAM1"></p>
+            <p id = "totalRAM2"></p>
+            <p id = "totalRAM3"></p>
+            <p id = "totalRAM4"></p>
+            <p id = "totalRAM5"></p>
+    		<br><br>
+    	Used RAM:
+            <p id = "usedRAM1"></p>
+            <p id = "usedRAM2"></p>
+            <p id = "usedRAM3"></p>
+            <p id = "usedRAM4"></p>
+            <p id = "usedRAM5"></p>
+    	</div>
+    </div>
+    	<br><br>
+
+        <script>
+        function loadDoc() {
+          var xhttp = new XMLHttpRequest();
+          xhttp.onreadystatechange = function() {
```

Comment:
All the dollar signs in the `.js` file indicate that the jQuery library is being used, it's been imported on line 8 of the this HTML file.

```html
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
```

jQuery is a pretty standard library to make alot of things simpler. One thing it does is provide a simplified wrapper around AJAX requests, I would definitely recommend using that.

Here is the documentation for making an AJAX post request with jQuery https://api.jquery.com/jquery.post/.